src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node head;
+    private Node tail;
+
+    public MyLinkedList() {
+        tail = null;
+        head = null;
+        size = 0;
```

Comment:
redundant. when you create an object of MyLinkedList, by default Java sets these default values.